### PR TITLE
Language corrections for install guides.

### PR DESCRIPTION
**Pull request by mpgreaves**

### CODE REVIEW

File: content/en/docs/guides/install-guides/_index.md [hunk 1]
```diff
@@ -120,31 +129,39 @@ sudo NEPHIO_DEBUG=false   \
...
+| *NEPHIO_HOME*             | path             | /home/$NEPHIO_USER | This is the directory into which the install scripts should be checked out.  |
+| *RUN_E2E*                 | false or true    | false              | This variable specifies whether or not end-to-end tests should be run.       |
+| *DOCKERHUB_USERNAME*      | alpha-num string |                    | This variable specifies the Docker Hub username.                             |
+| *DOCKERHUB_TOKEN*         | alpha-num string |                    | This variable specifies the password or token.                               |
+| *NEPHIO_REPO*             | URL              | https://github.com/nephio-project/test-infra.git | This variable specifies the URL of the         |
+|                           |                  |                    | repository to be used for installation                                       |
+| *NEPHIO_BRANCH*           | branch           | main/v3.0.0        | This variable specifies the tag or branch name to use in NEPHIO_REPO         |
+| *DOCKER_REGISTRY_MIRRORS* | List of URLs in  |                    | This variable specifies the list of Docker registry mirrors in JSON format.  |
```
Comment (efiacor) [inline content/en/docs/guides/install-guides/_index.md:144]: Table row formatting is thrown off by adding more | symbols.
Description for this and the K8S_CONTEXT below should be on the same row.

File: content/en/docs/guides/install-guides/_index.md [hunk 2]
```diff
@@ -120,31 +129,39 @@ sudo NEPHIO_DEBUG=false   \
...
+|                           | JSON format      |                    | If there are no mirrors to be set, then the variable remains empty. Here are |
+|                           |                  |                    | two example values: ``["https://docker-registry-remote.mycompany.com",       |
+|                           |                  |                    | "https://docker-registry-remote2.mycompany.com"]``                           |
```
Comment (efiacor) [inline content/en/docs/guides/install-guides/_index.md:147]: I think we have to add the dummy urls to the ignore list here - https://github.com/nephio-project/docs/blob/main/.linkspector.yml#L10

File: content/en/docs/guides/install-guides/_index.md (hunk 1 above)
Comment (mpgreaves) [reply content/en/docs/guides/install-guides/_index.md]: It has been corrected in [1b8b734](https://github.com/nephio-project/docs/pull/194/commits/1b8b734e41f33ae3691a0db612ebc7b640d896b2).

File: content/en/docs/guides/install-guides/_index.md (hunk 2 above)
Comment (mpgreaves) [reply content/en/docs/guides/install-guides/_index.md]: It appears that the build does not fail.

Comment (efiacor) [reply content/en/docs/guides/install-guides/_index.md]: Strange. It failed on the initial commit.
Looks good now.

**Pull request merged**